src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: 100vh;
+  background: url(../../img/header_bg--mobile&laptop.png) center / cover no-repeat;
```

Comment:
this image doesn't look good on the laptop
add media query and use another image for bigger screens (laptop, etc):
![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/157369ab-4131-4f2a-868a-06226df316b8)

![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/23d28aa0-c346-4522-ab20-c65976e3fbc2)